### PR TITLE
Revised for tutorial and example consistency

### PR DESCRIPTION
**Pull request by flywire**
Revised for tutorial and example consistency:

Change tutorial text to address any issues:
- no metadata
- source integrity check causing issues and was removed
- source changed to osm to remove compexities of using registered data
- consolidating `var` statements not described or included in example
- consolidating `.bindPopup` command statements not described or included in example

### CODE REVIEW

**Review by mourner — CHANGES_REQUESTED** [review]
I might address other changes later, but one thing to note is that OSM tile server is not used in the tutorials on purpose — the team behind OSM didn't want us to encourage using it on production websites, since it's not suited to that kind of load and it's expensive to keep. So I can't accept this particular change for sure. E.g. see here https://twitter.com/OSM_Tech/status/1308465290263629825

**Comment by flywire** [comment]
https://github.com/flywire/Leaflet/commits/patch-1 changed, I expected it would show here.

(Discussed at https://github.com/Leaflet/Leaflet/issues/7254#issuecomment-687609542.)

**Comment by flywire** [comment]
@mourner requested changes done with #92c8d25

File: example-basic.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (IvanSanchez) [inline example-basic.md:1]: I think you meant to edit files in `docs/`, but the changes here are *adding* files to the root (`/`) directory of the repo instead.

**Review by IvanSanchez — CHANGES_REQUESTED** [review]

**Comment by mourner** [comment]
-> #7816

**Pull request closed without merging**